langstream-cli/src/main/java/ai/langstream/cli/commands/applications/UIAppCmd.java

```diff
@@ -56,6 +57,12 @@ public class UIAppCmd extends BaseApplicationCmd {
     @CommandLine.Parameters(description = "Application ID")
     private String applicationId;
 
+    @CommandLine.Option(
+            names = {"-p", "--port"},
```

Comment:
what about "--ui-port" ? "--port" is too generic

Reply:
this command is litereally 
```
apps iu <appid>
```
I don't think it's confusing

In fact, in the docker run one, I used `--ui-port`